01_Jupyter_notebook_intro/index.md

```diff
@@ -85,6 +85,12 @@ Po úspěšné instalaci lze Jupyter Notebook spustit jednoduchým příkazem:
 $ jupyter-notebook
 ```
 
+Pokud tento příkaz čirou náhodou nefunguje, vyzkoušej následující:
+
+```shell
+$ python3 -m notebook
+```
+
```

Comment:
Ještě bych tu zopakoval to upozornění pro uživatele Windows.

Reply:
Díky. Místo opakování upozornění jsem tam právě dal to množné číslo v prvním upozornění, ale opakování je matkou moudrosti.